bentoml/_internal/bento/build_config.py

```diff
@@ -570,7 +653,8 @@ def __attrs_post_init__(self) -> None:
                     f"{self.docker.distro} does not supports conda. BentoML will only support conda with the following distros: {get_supported_spec('miniconda')}."
                 )
             if use_cuda and self.docker.distro not in get_supported_spec("cuda"):
-                raise BentoMLException(
+                # This is helpful when we support more CUDA version
```

Comment:
what does this comment mean?

Reply:
let me remove that this is not related.

Reply:
Unresolved this one too, are there commits pending?